homeassistant/components/dunehd/media_player.py

```diff
@@ -54,34 +68,36 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     )
 
 
-async def async_setup_entry(hass, config_entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
     """Add Dune HD entities from a config_entry."""
-    unique_id = config_entry.entry_id
+    unique_id: str = entry.entry_id
```

Comment:
`ConfigEntry` should be typed. You can verify this by adding `reveal_type(entry.entry_id)` here and running mypy, it'll print current type.
```suggestion
    unique_id = entry.entry_id
```